hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/WorkloadRegistryTest.java

```diff
@@ -31,6 +31,7 @@ public class WorkloadRegistryTest extends ZookeeperBaseTest {
 
     private static final String consumer1 = "consumer1";
     private static final String consumer2 = "consumer2";
+    private static final String clusterName = "primary-dc";
```

Comment:
same as above, what do You think about `new KafkaProperties().getClusterName()`?